### PR TITLE
Version bump nginx to 1.6.2.

### PR DESCRIPTION
**Pull request by isaacsu**
Main reason for this is to [keep up with Phusion Passenger 5](https://github.com/phusion/passenger/blob/dbd68e1be01aeb6e4dcd4a3a4e8f01991bc3c0fe/CHANGELOG#L170).

Also,
- Update cardboard to 2.1.0.
- Update tests to new `contain_class` rspec syntax.
- Update rvm version to ~~2.1.5~~ 2.0.0.

A newer version of the `highline` gem requires Ruby version >= 1.9.3. So, having `1.8.7` as one of the build targets doesn't work.

/cc @dgoodlad @fromonesrc @sandlerr

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ script: "./script/cibuild"
 gemfile: "this/does/not/exist"
 env: GITHUB_API_TOKEN=36a1b2a815b98d755528fa6e09b845965fe1e046 # token of user: some-public-token without any permissions to avoid api rate limit
 rvm:
-  - "1.8.7"
+  - "2.1.5"
```
Comment (tarebyte) [inline .travis.yml]: @isaacsu only we test versions of ruby that are on OSX by default, which means that we need `1.8.7` and `2.0.0`.

Can you update the `.travis.yml` to look like the one that [puppet-template](https://github.com/boxen/puppet-template/blob/master/.travis.yml) uses?

**Comment by staugaard** [comment]
Nice!

File: .travis.yml (hunk 1 above)
Comment (isaacsu) [inline .travis.yml]: >  to look like the one that puppet-template uses

I assume you mean having the `rvm` block look like this?
```
rvm:
  - "1.8.7"
  - "2.0.0"
```
One of the dependencies `highline` requires Ruby version >= 1.9.3 so `1.8.7` isn't going to work. Will having just `2.0.0` suffice?

**Comment by grosser** [comment]
:+1:

**Comment by jaredreynolds** [comment]
:+1: Any chance of getting this merged?  I'd like to get this tagged so I can then submit a PR to upgrade to 1.8.0.

**Comment by tarebyte** [comment]
This looks fine to me, @seanknox would there be a reason not to merge this in?

**Comment by seanknox** [comment]
:+1:

**Pull request merged**